### PR TITLE
[TASK] add svg viewhelper to theme_t3kit and register t3kit fluid nam…

### PR DESCRIPTION
**Pull request by pixelmatseriks**
…espaces

### CODE REVIEW

File: Classes/ViewHelpers/Render/SvgViewHelper.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        *
+        */
+        if (!file_exists($sourceAbs)) {
+            return 'no SVG file on /'.$source;
+        }
+        return $this->getInlineSvg($sourceAbs);
+    }
+
+    /**
+     * @param string $source
```
Comment (MattiasNilsson) [inline Classes/ViewHelpers/Render/SvgViewHelper.php]: Add comment for method

File: Classes/ViewHelpers/Render/SvgViewHelper.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+        if (\TYPO3\CMS\Core\Utility\GeneralUtility::isAllowedAbsPath($source)) {
+          return 'unable to open file';
+        }
+
+        $sourceAbs = \TYPO3\CMS\Core\Utility\GeneralUtility::getFileAbsFileName($source);
+        *
+        */
+        if (!file_exists($sourceAbs)) {
+            return 'no SVG file on /'.$source;
```
Comment (MattiasNilsson) [inline Classes/ViewHelpers/Render/SvgViewHelper.php]: Add space between string concatenation.

File: Classes/ViewHelpers/Render/SvgViewHelper.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    /**
+     * Generate a list from the content
+     *
+     * @param string $source
+     */
+    public function render($source)
+    {
+        $sourceAbs = PATH_site.$source;
+
+        /*
```
Comment (MattiasNilsson) [inline Classes/ViewHelpers/Render/SvgViewHelper.php]: Remove unused code.

File: Classes/ViewHelpers/Render/SvgViewHelper.php [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+     */
+    protected $escapeChildren = false;
+
+    /**
+     * @var boolean
+     */
+    protected $escapeOutput = false;
+
+
+    public function initializeArguments()
```
Comment (MattiasNilsson) [inline Classes/ViewHelpers/Render/SvgViewHelper.php]: Add PHPdoc

**Review by MattiasNilsson — CHANGES_REQUESTED** [review]
This looks good just some smaller PHPCGL issues.

**Review by dmh — APPROVED** [review]

**Comment by dmh** [comment]
@MattiasNilsson

**Comment by MattiasNilsson** [comment]
Looks good 👍

**Pull request merged**